frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/pipeline-step-utils.ts

```diff
@@ -35,10 +35,7 @@ const getMatchingStepDuration = (matchingStep?: TaskStatusStep) => {
   if (!matchingStep) return '';
 
   if (matchingStep.terminated) {
-    return calculateRelativeTime(matchingStep.terminated.startedAt);
-  }
-  if (matchingStep.running) {
-    return calculateRelativeTime(matchingStep.running.startedAt);
+    return calculateDuration(matchingStep.terminated.startedAt, matchingStep.terminated.finishedAt);
   }
```

Comment:
We should also show a duration if the pipeline step is running. This gives the user a creat indicator if its need 10%, 50% or 90% complete.

```suggestion
  if (matchingStep.terminated) {
    return calculateDuration(matchingStep.terminated.startedAt, matchingStep.terminated.finishedAt);
  }
  if (matchingStep.running) {
    return calculateDuration(matchingStep.running.startedAt, new Date() /* or make the second parameter optional and do this in calcDuration? */);
  }
```